Trappist/src/Promact.Trappist.Repository/Test/TestsRepository.cs

```diff
@@ -244,6 +244,7 @@ public TestsRepository(TrappistDbContext dbContext, IGlobalUtil util, IStringCon
                 var testCategoryList = await _dbContext.TestCategory.Where(x => x.TestId == testId).Include(x => x.Category).ToListAsync();
                 categoryListAc.ForEach(category =>
                 {
+                    category.QuestionCount = _dbContext.Question.Where(x => x.CategoryID == category.Id && x.CreatedByUserId == userId).ToList().Count();
```

Comment:
you can use count() method directly on where(). no need to do ToList(). adding ToList() increases overhead of firing query which count() will do by itself,